cmd/bbolt/command_check_test.go

```diff
@@ -2,32 +2,66 @@ package main_test
 
 import (
 	"bytes"
+	"fmt"
 	"io"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
 	main "go.etcd.io/bbolt/cmd/bbolt"
 	"go.etcd.io/bbolt/internal/btesting"
+	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
 func TestCheckCommand_Run(t *testing.T) {
-	db := btesting.MustCreateDB(t)
-	db.Close()
-	defer requireDBNoChange(t, dbData(t, db.Path()), db.Path())
-
-	rootCmd := main.NewRootCommand()
-	// capture output for assertion
-	outputBuf := bytes.NewBufferString("")
-	rootCmd.SetOut(outputBuf)
-
-	rootCmd.SetArgs([]string{
-		"check", db.Path(),
-	})
-	err := rootCmd.Execute()
-	require.NoError(t, err)
-
-	output, err := io.ReadAll(outputBuf)
-	require.NoError(t, err)
-	require.Equalf(t, "OK\n", string(output), "unexpected stdout:\n\n%s", string(output))
+	testCases := []struct {
+		name      string
+		args      []string
+		expErr    error
+		expOutput string
+	}{
+		{
+			name:      "check whole db",
+			args:      []string{"check", "path"},
+			expErr:    nil,
+			expOutput: "OK\n",
+		},
+		{
+			name:      "check valid pageId",
+			args:      []string{"check", "path", "--pageId", "3"},
+			expErr:    nil,
+			expOutput: "OK\n",
+		},
+		{
+			name:      "check invalid pageId",
+			args:      []string{"check", "path", "--pageId", "1"},
+			expErr:    guts_cli.ErrCorrupt,
+			expOutput: fmt.Sprintf("page ID (%v) out of range [2, 4)", 1),
```

Comment:
nit: this could be done without the formatting.

```suggestion
			expOutput: "page ID (1) out of range [2, 4)",
```

Reply:
done 👍🏽